compete.html

```diff
@@ -130,7 +130,7 @@ <h3>Building a robot</h3>
       <a name="signup"></a><h2>Want to compete?</h2>
       <div class="column m-8-12 m-offset-2-12 text-left">
         <p>
-          Entries for this year's competition are open until the 26th October 2018.
+          Applications for Student Robotics 2020 are open until around September 2019.
```

Comment:
This form signs them up to the master list, so isn't actually opening them to SR2020. The way we've done it in the past is signed people up to the master list now, and then in September email out an actual signup link, and update the one on the website.